### PR TITLE
Cash 1023 20 min loan res exp setup

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

File: src/components/Checkout/LoanReservation.vue [hunk 1]
```diff
@@ -89,38 +99,42 @@ export default {
...
+				const mins = differenceInMinutes(this.reservedDate.getTime(), Date.now());
+				const seconds = differenceInSeconds(this.reservedDate.getTime(), Date.now()) % 60;
 
-				let warningMessageUpperBoundMinutes = 6;
+				let warningMessageUpperBoundMinutes = 5;
 				let differenceInWords = `Reserved for ${mins} more minutes`;
 
 				if (this.activateTimer === true) {
-					warningMessageUpperBoundMinutes = 10;
+					warningMessageUpperBoundMinutes = 9;
```
Comment (mcstover) [inline src/components/Checkout/LoanReservation.vue]: Previously these 2 bound settings were 1 minute off due to the inclusion of seconds. In other words 6:59 and 10:59 would flip the class to red whereas we want it to flip at 5:59 + 9:59.

File: src/components/Checkout/LoanReservation.vue [hunk 2]
```diff
@@ -89,38 +99,42 @@ export default {
...
-				if ((reservedDate.getTime() - Date.now()) <= 0 || this.isExpiringSoon) {
+				if ((this.reservedDate.getTime() - Date.now()) <= 0 || this.isExpiringSoon) {
 					clearInterval(this.reservationMessageId);
 					this.loanReservationMsg1 = true;
 					this.loanReservationMsg2 = false;
 					this.loanReservationMsg3 = false;
 					this.loanReservationMsg4 = false;
 				} else if (mins > warningMessageUpperBoundMinutes) {
 					this.setDifferenceInWords(differenceInWords);
+					this.loanReservationMsg1 = false;
```
Comment (mcstover) [inline src/components/Checkout/LoanReservation.vue]: reassign prior messages to false... goes for those below too. This along with the markup refactor prevents multiple loan reservation statements from showing up.

File: src/util/experimentPreFetch.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ let activeExperiments = [
 	'pinned_filter',
 	'algolia_search',
 	'expandable_loan_cards',
+	'loan_res_20',
```
Comment (mcstover) [inline src/util/experimentPreFetch.js]: global b/c it's used everywhere the AddToBasketInterstitial exists AND in Checkout

File: src/components/Checkout/LoanReservation.vue [hunk 4]
```diff
@@ -89,38 +99,42 @@ export default {
...
 					this.loanReservationMsg3 = true;
 				} else if (mins <= 1) {
 					differenceInWords = this.activateTimer === true
 						? `Reservation expires in ${seconds}s`
 						: 'Reserved for 1 more minute';
 
 					this.setDifferenceInWords(differenceInWords);
+					this.loanReservationMsg3 = false;
 					this.loanReservationMsg4 = true;
 				}
```
Comment (BoulderBrains) [inline src/components/Checkout/LoanReservation.vue]: Nice refactor in this file!

File: src/components/Checkout/LoanReservation.vue (hunk 1 above)
Comment (ksekhri) [reply src/components/Checkout/LoanReservation.vue]: Nice

**Pull request merged**